docs/source/datasetinsights.evaluation_metrics.rst

```diff
@@ -18,10 +18,10 @@ datasetinsights.evaluation\_metrics.average\_precision
    :undoc-members:
    :show-inheritance:
 
-datasetinsights.evaluation\_metrics.average\_precision\_2d\_bbox
-----------------------------------------------------------------
+datasetinsights.evaluation\_metrics.average\_precision\_2d
+----------------------------------------------------------
 
-.. automodule:: datasetinsights.evaluation_metrics.average_precision_2d_bbox
+.. automodule:: datasetinsights.evaluation_metrics.average_precision_2d
    :members:
```

Comment:
Have you renamed the module from average_precision_2d_bbox to average_precision_2d? btw what's the reason for this change?

Reply:
It was changed as part of this [PR](https://github.com/Unity-Technologies/datasetinsights/pull/20). The docs was not updated accordingly.